ansible/inventory.yaml

```diff
@@ -29,5 +37,12 @@ all:
 
         # HammerDB Setup
         hammerdb_version: 4.12
-        hammerdb_base_path: /opt/HammerDB
-        oracle_client_home: /usr/lib/oracle/19.26/client64
\ No newline at end of file
+        hammerdb_base_path: "{{ benchmark_base_path }}/hammerdb"
+        hammerdb_install_path: "{{ hammerdb_base_path }}/{{ hammerdb_version }}"
+
+        # SwingBench Setup (Unix/Linux/Windows JDK 11, Updated November 4th 2023)
```

Comment:
WDYT keeping both HamerDB and swing bench installation separeted each other. including inventory and configuration. WDYT ?

Reply:
That makes sense. I will make necessary changes.

Reply:
We refer to this repo with HammerDB in the documentation, so I think we better to keep HammerDB as the default installation. 
Swing-bench another good tool but I think let's make it optional at the moment. something I am also not aware how it works and how is useful or what advantage it has.

Reply:
updated to hammerdb as default option.